CHANGELOG.md

```diff
@@ -13,12 +13,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `UpdateListener` now has an associated type `Err` instead of a generic
 - `AsUpdateStream` now has an associated type `StreamErr` instead of a generic
 - Rename `dispatching::stop_token::{AsyncStopToken, AsyncStopFlag}` => `stop::{StopToken, StopFlag}`
+- Replace the generic error type `E` with `RequestError` for REPLs (`repl(_with_listener)`, `commands_repl(_with_listener)`)
 - The following functions are now `#[must_use]`:
   - `BotCommands::ty`.
   - `CommandDescriptions::{new, global_description, username, username_from_me}`.
   - `teloxide::filter_command`.
   - `teloxide::dispatching::dialogue::enter`.
 
+### Added
+
+- `requests::ResponseResult` to `prelude`
```

Comment:
Do you think we need this?

Reply:
Yes, we use it in some simple examples such as `command.rs` and `admin.rs`. Since `respond` is already in the prelude, I think `ResponseResult` must be also there.